src/scripts/main.js

```diff
@@ -21,7 +21,26 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  function buildList(obj) {
+    const ul = document.createElement('ul');
+
+    for (const key in obj) {
+      const li = document.createElement('li');
+      li.textContent = key;
+
+
+      if (Object.keys(obj[key]).length > 0) {
+        li.appendChild(buildList(obj[key]));
+      }
+
+      ul.appendChild(li);
+    }
+
+    return ul;
+  }
```

Comment:
This `buildList` function is nested inside the `createTree` function. While this is not necessarily a problem, it can make the code harder to read and understand. Consider moving it outside of `createTree` and passing any necessary parameters to it. This will also make it easier to test this function separately.